client/src/components/survey/SurveyPage3.jsx

```diff
@@ -80,6 +117,9 @@ const SurveyPage3 = props => {
             <Button onClick={submitSurveyPage3} color="secondary" className={classes.continueButton}>
               SUBMIT
             </Button>
+            <Button onClick={openSurveyPage4} color="secondary" className={classes.continueButton}>
+              CONTINUE TO BEHAVIORAL SURVEY
+            </Button>
```

Comment:
Should we not just have this be an additional page in our survey i.e. have page 3 -> page 4 and then move the SUBMIT option to page 4? This could potentially confuse the user

Reply:
I wanted to wait for the feedback as to confirm whether plugging in the behavioral survey at this page 3 is optimal. Since people who know UX might want to change page 3 directly to behavioral component. So that is why I left the button there. We can remove it once it has been tested?

Reply:
I'm not sure I follow - we should be sure this is working already before merging in any changes. We should loop in @AdhamAH or @BrianHHough but I think going through the full survey, so now pages 1->4, having the submit button only on page 4, would be optimal.

Or, maybe simply reorder the pages by renaming them, so that page 3 is this behavioral section, and page 4 is the "additional comments + submit"